fenix/app/src/main/res/values/strings.xml

```diff
@@ -2058,6 +2058,156 @@
     <!-- Snackbar button text to navigate to telemetry settings.-->
     <string name="experiments_snackbar_button">Go to settings</string>
 
+    <!-- Review quality check feature-->
+    <!-- Name for the review quality check feature used as title for the panel. -->
+    <string name="review_quality_check_feature_name">Review checker</string>
+    <!-- Description for grades A and B for review quality check adjusted grading. -->
+    <string name="review_quality_check_grade_a_b_description">Reliable reviews</string>
+    <!-- Description for grade C for review quality check adjusted grading. -->
+    <string name="review_quality_check_grade_c_description">Mix of reliable and unreliable reviews</string>
+    <!-- Description for grades D and F for review quality check adjusted grading. -->
+    <string name="review_quality_check_grade_d_f_description">Unreliable reviews</string>
+    <!-- Headline for review quality check review reliability card. -->
+    <string name="review_quality_check_grade_title">How reliable are these reviews?</string>
+    <!-- Headline for review quality check adjusted rating card. -->
+    <string name="review_quality_check_adjusted_rating_title">Adjusted rating</string>
+    <!-- Description for review quality check review reliability card. -->
+    <string name="review_quality_check_adjusted_rating_description">Unreliable reviews removed</string>
+    <!-- Headline for review quality check highlights card. -->
+    <string name="review_quality_check_highlights_title">Highlights from recent reviews</string>
+    <!-- Headline for review quality check explanation card. -->
+    <string name="review_quality_check_explanation_title">How we determine review quality</string>
+    <!-- Paragraph from review quality check explanation card explaining reliability. First parameter is the Fakespot product name. -->
+    <string name="review_quality_check_explanation_body_reliability">We use AI technology from %s by Mozilla to analyze the reliability of product reviews. This analysis will only help you assess review quality, not product quality.</string>
+    <!-- Paragraph from review quality check explanation card explaining review grading. First parameter is for highlighted string defined in review_quality_check_info_review_grade_header_highlighted. -->
+    <string name="review_quality_check_info_review_grade_header" tools:ignore="UnusedResources">We assign each product’s reviews a %s from A to F.</string>
```

Comment:
Again let's explain this grading system.
Also, why is "letter grade" not just part of this string and instead replaced by a parameter? 🤔